contrib/kubernetes/flowlogs-pipeline.conf.yaml

```diff
@@ -255,7 +256,10 @@ parameters:
       - name: bandwidth_per_network_service
         type: counter
         filter:
-          key: name
+          key: ""
+          value: ""
```

Comment:
Since `filter` is being deprecated, shouldn't we just delete this section?

Reply:
done